beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -270,6 +270,12 @@ type produceBlockResponseV2Json struct {
 	Data    *beaconBlockContainerV2Json `json:"data"`
 }
 
+// produceBlindedBlockResponseJson is used in /v1/validator/blinded_blocks/{slot} API endpoint.
+type produceBlindedBlockResponseJson struct {
```

Comment:
What's the difference between `bellatrixProduceBlindedBlockResponseJson`?

Reply:
`produceBlindedBlockResponseJson` includes a `Data` field, which itself has 3 fields:
- `Phase0Block`
- `AltairBlock`
- `BellatrixBlock`
This is because we use a `oneof` in protobuf. The purpose of `bellatrixProduceBlindedBlockResponseJson` is to contain only one field - the bellatrix block.